src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
I think these(CREATED_POST, SAVED_POST, REACT_POST) are a "relation" between user and item.
They seem more belong to user api